charts/quickwit/values.yaml

```diff
@@ -93,6 +93,7 @@ searcher:
 
   podAnnotations: {}
 
+  # Deprecated, please use searcher.service.annotations
   serviceAnnotations: {}
```

Comment:
Do you think we can expect a lot of other configurability feature requests for the service object? if not, I don't think it is worth deprecating `serviceAnnotations`. We can keep it as is along with a new `serviceType`.

Reply:
Don't think so! (at least for now), I agree!